### PR TITLE
Add flags to mix task 

### PR DESCRIPTION
**Pull request by mgutierrezerpi**

### CODE REVIEW

**Comment by mgutierrezerpi** [comment]
Test if the version changes too (not only if it's not there)

File: lib/mix/tasks/scapa.gen.versions.ex [hunk 1]
```diff
@@ -1,11 +0,0 @@
-defmodule Mix.Tasks.Scapa.Gen.Versions do
```
Comment (brunvez) [inline lib/mix/tasks/scapa.gen.versions.ex:1]: Don't delete this, they are supposed to be different use cases

File: lib/scapa/cli.ex [hunk 2]
```diff
@@ -9,12 +9,12 @@ defmodule Scapa.CLI do
...
   @doc version: "124861924"
   def generate_versions(files_pattern) do
-    files_to_versionate(files_pattern)
+    files_to_versioning(files_pattern)
     |> Enum.map(&{&1, add_versions_to_file(&1)})
     |> Enum.filter(&elem(&1, 1))
   end
 
-  defp files_to_versionate(files_pattern) do
+  defp files_to_versioning(files_pattern) do
```
Comment (brunvez) [inline lib/scapa/cli.ex:17]:
```suggestion
  defp files_to_version(files_pattern) do
```

File: lib/scapa/options.ex [hunk 3]
```diff
@@ -0,0 +1,10 @@
+defmodule Scapa.Options do
```
Comment (brunvez) [inline lib/scapa/options.ex:1]:
```suggestion
defmodule Scapa.CLI.Options do
```

**Pull request closed without merging**